README.md

```diff
@@ -239,13 +241,26 @@ Here we can set the 'host' port, the port which docker forwards into the contain
 ####volumes
 This allows you to mount host volumes in the container.
 
-```yaml        
+```yaml
 	        volumes:
 	          es_vol:
 	            host: /mnt/es_data
 	            container: /data
 ```
 
+
+####links_restarts
+If set to True, the init service jobs are set up to restart and re-link if the linked-to containers restart.
+
+####links
+This allows the container to be linked to other containers. The first entry in each key is the external container name, the second is the internal name for the container.
+
+```yaml
+	        links:
```

Comment:
there's a tab-vs-space indentation inconsistency here which causes the hierarchy to be rendered wrongly.